docs/sdk/tutorials/export_a_kili_project.md

```diff
@@ -0,0 +1,207 @@
+* Depending on the export type, it downloads the asset media:
+     * Kili: ❌
+     * YOLO: ❌
+     * COCO: ✅
+* Depending on the `export_type` argument, it will export the latest label or all the labels.
+* Depending on the `split_option` argument, the layout will be one folder for all the jobs, or one folder per job.
+* The `single_file` argument tells if the labels data should be exported into one single file. Note that some formats are single_file only, and that some others cannot be output into a single file.
+     * Kili: single file or multiple files.
+     * YOLO: multiple files only.
+     * COCO: single-file only.
+
+For all the formats, in the output archive, a README.kili.txt file is also output. Here is an example of its contents:
+```
+Exported Labels from KILI
+=========================
+
+- Project name: Awesome annotation project
+- Project identifier: abcdefghijklmnop
+- Project description: This project contains labels, most of which are awesome.
+- Export date: 20221125-093324
+- Exported format: kili
+- Exported labels: latest
+```
+
+
+### Kili format, one file per asset
+```python
+from kili.client import Kili
+kili = Kili()
+kili.export_labels(
+    project_id = "<your_project_id>",
+    filename = "/tmp/export.zip",
+    fmt = "kili",
+)
+```
+will export all the assets payloads and the associated labels, with one json file per asset, into the `/tmp/export.zip` folder.
+
+### Kili format, one file for the whole project
+```python
+from kili.client import Kili
+kili = Kili()
+kili.export_labels(
+    project_id = "<your_project_id>",
+    filename = "/tmp/export.zip",
+    fmt = "kili",
+    single_file = True,
+)
+```
+will export all the assets payloads and the associated labels, with one json file per asset, into the `/tmp/export.zip` folder.
+
+### YOLO formats
+You can also export to the yolo format, when you have at least one Object Detection job with bounding boxes. You can select the `"yolo_v4"`, `"yolo_v5"` or `"yolo_v7"`. The difference between each format is the structure of the metadata YAML file, which specifies the object classes. In all the cases, it produces one file per asset in the Yolo format, containing the last DEFAULT or REVIEW label that has been produced. Each YOLO label has the following shape:
+```
+2        0  123 33 145
+^        ^  ^   ^  ^
+class    x  y   w  h
+```
+where:
+
+   * `class` is the class index in the classes list contained in the YOLO metadata file.
+   * `x` is the x-coordinate relative to the image width (between 0.0 and 1.0) of the center of the bounding box.
+   * `y` is the y-coordinate relative to the image height (between 0.0 and 1.0) of the center of the bounding box.
+   * `w` is the width relative to the image width (between 0.0 and 1.0) of the bounding box.
+   * `h` is the height relative to the image height (between 0.0 and 1.0) of the bounding box.
+
+Here is how to call the YOLO export (YOLO V5 here):
```

Comment:
I'd rephrase. Call is pretty code-centric. How about: "Here's how to export to YOLO (in this example, YOLOv5):"